Scal 878 block charts with string parameter

laganows opened the pull request:
Block charts for display when string type parameter given, escape parameters with escapeHtml() function and handle message error when empty results for experiment (based on moes_info from EM).

laganows committed:
SCAL-878-Block charts (lindev, pareto, interaction, dendogram) for plot with string parameter

laganows committed:
Merge branch 'master' into SCAL-878-Block-charts-with-string-parameter

laganows committed:
SCAL-878-merged with newest master && escape parameters in js

laganows committed:
SCAL-878-merged with newest master && escape parameters in js

laganows committed:
SCAL-878: Added functions to check plot possibility
 Added functions to check plot possibility (based on moes_info from EM) and escape parameters with escapeHtml() function in modals

laganows committed:
SCAL-878: Merged with newest development

laganows committed:
Merge remote-tracking branch 'remotes/origin/development' into SCAL-878-Block-charts-with-string-parameter

laganows committed:
SCAL-878: Blocked DE charts for string parameter
Blocked DE charts for string parameter. Deleted function to check if there is no results to plot (this is not this task) and refactored code for every modal (except dendogram) in this issue.

kliput commented on app/visualisation_methods/dendrogram/_modal.html.haml:
```diff
@@ -62,6 +62,15 @@
       var selected_parameters = selectedFiles.toArray();
       var url = "#{@prefix}/chart_instances/dendrogram";
+      for (i = 1; i <= selected_parameters.length; i++) {
+        escapeHtml(selected_parameters[i-1].trim())
```
Co robi ta linijka? Funkcja escapeHtml zwraca wartość, która nie jest nigdzie używana, a parametry nie są escapowane.

kliput commented on app/visualisation_methods/interaction/_modal.html.haml:
```diff
@@ -69,6 +69,14 @@
       var x_axis = $("#interactionModal select#firstParam option:selected").val();
       var y_axis = $("#interactionModal select#secondParam option:selected").val();
       var outputParam = $("#interactionModal select#outputParam option:selected").val();
+      escapeHtml(x_axis)
```
jw. - escaleHtml używa się w inny sposób

kliput commented:
Trzeba przetestować escapowanie html

laganows committed:
SCAL-878: Fixed usage of escapeHtml in modals

laganows committed:
SCAL-878: Fixed usage of escapeHtml in 3D modal

laganows committed:
Merge branch 'development' into SCAL-878-Block-charts-with-string-parameter

Conflicts:
	app/visualisation_methods/three_d/_modal.html.haml

laganows committed:
SCAL-878: Block charts for string parameter && escape html
 Block charts (dendrogram, interaction, kmeans, lindev, pareto, three_d) for string parameter  and escape html in modals
